sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -393,6 +398,18 @@ def test_global_fanout(self):
           | beam.CombineGlobally(combine.MeanCombineFn()).with_fanout(11))
       assert_that(result, equal_to([49.5]))
 
+  @attr('ValidatesRunner')
```

Comment:
Why is this validates runner?

Reply:
When should we use validates runner and when should not?